server/src/main/java/org/opensearch/extensions/DiscoveryExtensionNode.java

```diff
@@ -82,6 +82,14 @@ public List<ExtensionDependency> getDependencies() {
         return dependencies;
     }
 
+    public boolean dependenciesContain(ExtensionDependency dependency) {
+        for (ExtensionDependency extensiondependency : this.dependencies) {
+            return dependency.getUniqueId().equals(extensiondependency.getUniqueId())
```

Comment:
This can return `false` as well, are we fine with it? If not, then if condition was the right way to go about it.

Reply:
Since the return type is `boolean`, We should return either `true` or `false`. Let's keep it the old way.